web/apps/osee/src/app/ple/artifact-explorer/lib/components/hierarchy/artifact-operations-context-menu/artifact-operations-context-menu.component.html

```diff
@@ -16,12 +16,17 @@
 			<button
 				mat-menu-item
 				[disabled]="!_branchEditable()"
-				(click)="selectOption(operationType)">
+				(click)="selectOption(operationType)"
+				class="tw-flex tw-w-full tw-items-center tw-overflow-hidden tw-text-ellipsis tw-whitespace-nowrap tw-px-4 tw-py-2 tw-text-left">
 				<mat-icon
 					[class]="getIconClasses(operationType.materialIcon)"
+					class="tw-mr-2"
```

Comment:
Can we use padding or something else instead?

Reply:
done